cit-api/pipeline/management/commands/bootstrap.py

```diff
@@ -22,7 +22,7 @@ def handle(self, *args, **options):
         non_bca_resources = DataSource.objects.exclude(name__in=[
             'bc_assessment_economic_region', 'bc_assessment_census_subdivision',
             'bc_assessment_regional_district'
-        ]).order_by('import_order')
+        ]).filter(import_order=44).order_by('import_order')
```

Comment:
Do we need this filter? I know most of us won't hit it anyways.

Reply:
Right I forgot to remove it cause I was only testing my new data source